pxtsim/runtime.ts

```diff
@@ -1054,6 +1087,47 @@ namespace pxsim {
             c.start = 0;
             c.numstops++;
         }
+
+        // Wrapper for the setTimeout
+        schedule(fn: () => void, timeout: number): number {
+            // We call the timeout function and add its id to the timeouts scheduled.
+            if (timeout <= 0) return -1;
+            let id = setTimeout(fn, timeout);
+            runtime.timeoutsScheduled.push(new TimeoutScheduled(id, fn, timeout, Date.now()));
```

Comment:
we use U.now() in other places.

Reply:
I've seen both around, is the Util one preferred by you?

Reply:
yes use U.now()